content/terraform/terraform.tf

```diff
@@ -103,3 +104,12 @@ module "visit_us_listener" {
   priority = "101"
   path = "/visit-us"
 }
+
+module "installations_listener" {
+  source = "../../shared-infra/terraform/service_alb_listener"
+  alb_listener_https_arn = "${local.alb_listener_https_arn}"
+  alb_listener_http_arn = "${local.alb_listener_http_arn}"
+  target_group_arn = "${module.content.target_group_arn}"
+  priority = "110"
```

Comment:
This is 110 to account for https://github.com/wellcometrust/wellcomecollection.org/commit/1ac618f3d44ea4b5ef1cca82fe9a659968ed873a#diff-f1b06eb5cc7af606f6d9cc85b872b912